responsibleai/responsibleai/rai_insights/rai_insights.py

```diff
@@ -471,6 +473,19 @@ def _validate_rai_insights_input_parameters(
                     columns=[target_column], axis=1)
                 small_test_data = small_test_data.drop(
                     columns=[target_column], axis=1)
+                if len(small_train_data.columns) == 0 or \
+                        len(small_test_data.columns) == 0:
+                    if has_dropped_features:
+                        raise UserConfigValidationException(
+                            'All features have been dropped from the dataset.'
+                            ' Please do not drop all the features'
```

Comment:
(nit) missing `.` at the end of the sentence.

Reply:
added